src/front-end/typescript/lib/pages/opportunity/team-with-us/lib/components/form.tsx

```diff
@@ -610,6 +666,17 @@ export function isValid(state: Immutable<State>): boolean {
   );
 }
 
+/**
+ * Gets the numerical value of a select field.
+ *
+ * @param state
+ * @returns number | null
+ */
+export function getNumberSelectValue(state: Immutable<Select.State>) {
+  const value = parseFloat(Select.getValue(state));
+  return isNaN(value) ? null : value;
+}
```

Comment:
I recommend adding a return type to avoid using type inference. Reduces risk of bugs in the future when refactoring.